### DISCUSSION TITLE
GBA Netplay

### DISCUSSION THREAD

**Discussion by Amendoleaa** [OnionUI/Onion#1150] [Q&A] [answered]
Hello everyone :) 
I ask to programmers and collaborators if there will be one day the multiplayer play function on GBA roms with WiFi as you made with GB and GBC (for example, as the android app MyBoy).
Thanks in advance for all responses here below <3

**Comment by XK9274** [comment, marked as answer]
I think mGBA will be the closest contender for GBA netplay, you can view the tasklist here:

https://github.com/mgba-emu/mgba/wiki/Task-List

Looks like some work will be done in mGBA 0.11 and some in mGBA 1.0 (but could be a long way away😔).. Also whether that goes in the core or not who knows 

We can dream!

**Reply by Amendoleaa** [reply]
Thank you very much <3

**Reply by hitsize** [reply]
Does this take a long time? I am so waiting. The ecosystem called GBA has software for most games, and the moment it becomes multiplayer, you will be able to share happiness and memories with many people. I sincerely hope that gba multiplayer will be supported soon.
